### DISCUSSION TITLE
Callouts/speech bubbles

### DISCUSSION THREAD

**Discussion by luke777** [hakimel/reveal.js#3370] [Q&A]
I am new to reveal.js.  I would like to make a presentations that describes how to use a GUI.  I was thinking of including a series of screenshots.  I would like to include something like powerpoint callouts/speech bubbles to describe different parts to the user interface.  I could make the presentation in powerpoint, then save the slides as images.  Then include the images in a reveal.js presentations, however, this makes editing a bit awkard.  I wondered if there is a way to acheive the same results in reveal.js  Any pointers or examples appreciated.

**Comment by t-fritsch** [comment]
Hi
In fact you can display anything you want over your images, in your situation you could add some `<div>` elements in your slides : 
- with the text you wand inside
- with a styling you like (there are many css examples out there to produce callouts/speech bubbles out there) 
- positioned with css styles `position:absolute;top:...,left:...`

Top and left position may require to iterate until you find the right values (livereload server may ease the process : https://revealjs.com/installation/#full-setup )

If you're not into HTML and CSS, maybe you could take a closer look at https://slides.com/ (commercial product that uses reveal.js, it is also made by the author of reveal @hakimel 😉) : I never used it but it allows to build presentations with a wysiwyg editor and there's a free plan I think
